frontend/src/components/Forms/AddIncidentForm.js

```diff
@@ -1,126 +1,181 @@
-    <Row gutter={16}>
-      <Col>
-        <Form.Item>
-          <Field
-            id="incident_timestamp"
-            name="incident_timestamp"
-            label="Incident Date and Time*"
-            placeholder="Please select date and time"
-            component={RenderDate}
-            showTime
-            validate={[required, dateNotInFuture]}
-          />
-        </Form.Item>
-        <Form.Item>
-          <Field
-            id="incident_description"
-            name="incident_description"
-            label="Incident Description*"
-            placeholder=""
-            component={RenderAutoSizeField}
-            validate={[required]}
-          />
-        </Form.Item>
-        <Form.Item>
-          <Field
-            id="reported_timestamp"
-            name="reported_timestamp"
-            label="Reported Date and Time"
-            placeholder="Please select date and time"
-            component={RenderDate}
-            showTime
-          />
-        </Form.Item>
-        <Form.Item>
-          <Field
-            id="reported_by"
-            name="reported_by"
-            label="Reporter's Name"
-            placeholder=""
-            component={RenderField}
-          />
-        </Form.Item>
-        <Form.Item>
-          <Field
-            id="reported_by_role"
-            name="reported_by_role"
-            label="Job Title of Reporter"
-            placeholder=""
-            component={RenderField}
-          />
-        </Form.Item>
-        <Form.Item>
-          <Field
-            id="followup_type_code"
-            name="followup_type_code"
-            label="EMPR Action*"
-            component={renderConfig.SELECT}
-            data={props.followupActionOptions}
-            validate={[required]}
-          />
-        </Form.Item>
-        <Form.Item>
-          <Field
-            id="followup_inspection_no"
-            name="followup_inspection_no"
-            label="NRIS Inspection Number"
-            placeholder=""
-            component={RenderField}
-          />
-        </Form.Item>
-        <Form.Item>
-          <Field
-            id="closing_report_summary"
-            name="closing_report_summary"
-            label="Closing Report Summary"
-            placeholder=""
-            component={RenderAutoSizeField}
-          />
-        </Form.Item>
-      </Col>
-    </Row>
-    <div className="right center-mobile">
-      <Popconfirm
-        placement="topRight"
-        title="Are you sure you want to cancel?"
-        onConfirm={props.closeModal}
-        okText="Yes"
-        cancelText="No"
-      >
-        <Button className="full-mobile" type="secondary">
-          Cancel
-        </Button>
-      </Popconfirm>
-      <Button className="full-mobile" type="primary" htmlType="submit" disabled={props.submitting}>
-        {props.title}
-      </Button>
-    </div>
-  </Form>
-);
+class AddIncidentForm extends Component {
+  constructor(props) {
```

Comment:
I don't believe this constructor method is required

Reply:
The value of the determination dropdown is put in the state and is used to conditionally render the sub-paragraph component. The value comes in through the initialValues. The state has to be set here in the constructor, else when the window opens and the determination is "DO", the state is empty and the component is not rendered.